### ISSUE TITLE
When will foundation 7 be released.

### ISSUE THREAD

**Issue by drnkwati** [foundation/foundation-sites#12514]
Is foundation 7 still in the works as promised two years ago in the roadmap. Should we consider the project suspended? I love foundation but I am really concerned about its future. 
🤔

**Comment by ameotoko** [comment]
Seems to be dead. Or at least not maintained.

**Comment by maietta** [comment]
Not dead. Just slow.

**Comment by ameotoko** [comment]
PRs are not reviewed since July.

**Comment by wallacio** [comment]
https://github.com/foundation/foundation-sites/discussions/12501#discussioncomment-4381419

**Comment by ameotoko** [comment]
Yes, I’ve seen that. Does it mean that F6 will not get fixes anymore? Should we just wait for v7?

**Comment by ms-carterk** [comment]
any new news?

**Comment by eydolan** [comment]
anything?

**Comment by ameotoko** [comment]
Foundation seems to be meeting the same fate as SemanticUI.

**Comment by maietta** [comment]
I think it's one guy doing everything.

People need to donate or otherwise chip in to keep projects they care about, going.

I don't like that Foundation uses jQuery.

I was thinking about building a component library and layout system using Tailwind to replicate Foundation but of those I talked with, conversations went dead.

People are moving on, and it's a shame.

With modern js frameworks, Foundation as it is now, doesn't fit into the ecosystems well.

If there is to be a version 7 or 8, it's going to have to take people actually doing something other than complaining.

@joeworkman I'm sure makes nothing on this and has to volunteer his time to keep it moving.

**Comment by ameotoko** [comment]
> it's going to have to take people actually doing something other than complaining

What exactly do you want people to do? With pull requests being ignored for years, there's very little one can do 🤷🏻‍♂️. Last commit on development branch was 9 months ago. Last PR merged almost 2 years ago.

> I was thinking about building a component library and layout system using Tailwind to replicate Foundation but of those I talked with, conversations went dead.

IMO Tailwind is good as it is. I'm actually currently migrating my project to Tailwind from Foundation.

> @joeworkman I'm sure makes nothing on this and has to volunteer his time to keep it moving

I have an impression that he uses Foundation in one of his commercial products, which probably was the main reason for taking Foundation over.

**Comment by drnkwati** [comment]
@ameotoko You should be grateful that people actually care about foundation and expect regular updates. This indicates that a community of developers are committed to the project success and have choose foundation over hundreds of other frameworks. If the framework is dead, simply archive it and notify the community. I personally detest the fact that jQuery still exists within foundation as such turned to other solutions. Disrespecting contributors who have sacrificed valuable time is a Nono!
I suggest that the project should be archived if development has halted.

**Comment by maietta** [comment]
> > it's going to have to take people actually doing something other than complaining
> 
> What exactly do you want people to do? With pull requests being ignored for years, there's very little one can do 🤷🏻‍♂️. Last commit on development branch was 9 months ago. Last PR merged almost 2 years ago.
> 
> > I was thinking about building a component library and layout system using Tailwind to replicate Foundation but of those I talked with, conversations went dead.
> 
> IMO Tailwind is good as it is. I'm actually currently migrating my project to Tailwind from Foundation.
> 
> > @joeworkman I'm sure makes nothing on this and has to volunteer his time to keep it moving
> 
> I have an impression that he uses Foundation in one of his commercial products, which probably was the main reason for taking Foundation over.

I make less than $300/mo. USD and despite 2 years working on another source project, couldn't get a single donation of any amount, otherwise I would have considered forking this project and running with it. Did manage to get $20, but all the people I spoke with who previous expressed an interest in discussing the project, all failed to discuss the project with me. I abandoned my efforts because people don't' seem to be all that serious.

So in summary, I tried to take a lead, but without those who follow, I'm not the right leader. I've done my part.

**Comment by drnkwati** [comment]
Hi @joeworkman this [roadmap](https://github.com/foundation/foundation-sites/wiki/Project-Roadmap) has not been followed and you have made no comments about this topic. Should we assume the project will be archived without further development. Thanks

**Comment by joeworkman** [comment]
I literally just released 6.9 on Friday. Yes. I have not been able to follow the goal that I wanted. I've had to focus on commercial projects. However, the project is not abandoned. I do not appreciate the tone especially after the work that I put into 6.9.
